core/src/node/config.rs

```diff
@@ -57,6 +57,8 @@ pub struct NodeBaseConfig {
     pub archive_block_provider: ArchiveBlockProviderConfig,
 
     pub blockchain_block_provider: BlockchainBlockProviderConfig,
+
+    pub single_node: bool,
```

Comment:
Node base does not know anything about mempool or collator. We should probably just remove `wait_for_neighbours` from `init` and call it explicitly